modules/rostests/kmtests/kmtest/kmtest.c

```diff
@@ -1,8 +1,9 @@
 /*
- * PROJECT:         ReactOS kernel-mode tests
- * LICENSE:         GPLv2+ - See COPYING in the top level directory
- * PURPOSE:         Kernel-Mode Test Suite Loader Application
- * PROGRAMMER:      Thomas Faber <thomas.faber@reactos.org>
+ * PROJECT:     ReactOS kernel-mode tests
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     Kernel-Mode Test Suite Loader Application
+ * COPYRIGHT:   Copyright 2011 Thomas Faber <thomas.faber@reactos.org>
+ *              Copyright 2018 Serge Gautherie <reactos-git_serge_171003@gautherie.fr>
```

Comment:
Adding 2 lines, and already adding copyright?

Reply:
Why not? I can remove this line if unwanted...